src/config/rcxml.c

```diff
@@ -400,6 +420,12 @@ entry(xmlNode *node, char *nodename, char *content)
 		wl_list_append(&rc.workspace_config.workspaces, &workspace->link);
 	} else if (!strcasecmp(nodename, "popupTime.desktops")) {
 		rc.workspace_config.popuptime = atoi(content);
+	} else if (!strcasecmp(nodename, "id.region.regions")) {
+		current_region = znew(*current_region);
+		wl_list_append(&rc.regions, &current_region->link);
+		fill_region_geo(nodename, content);
```

Comment:
`region` elements are parsed differently to keybinds, mousebinds and libinput devices.
Consider setting an `in_region` bool in `xml_tree_walk()` and `zalloc()` on hitting id/name.
I guess either will work fine, but the recursive `xml_tree_walk()` is more consistent.

Reply:
done